### PR TITLE
Red Screen

### PR DESCRIPTION
**Pull request by AlbertVVila**
When the player is hit by an enemy the borders of the screen turns red.

### Test

- For better screen display test it in GameBuild or GameBuild Debug
- Get hit by an enemy and check that the screen turns red
- Check that resizing the screen the borders stay anchored to screen size.

### CODE REVIEW

File: Scripts/DamageFeedbackUI/DamageFeedbackUI.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#else
+#define DamageFeedbackUI_API __declspec(dllimport)
+#endif
+
+class DamageFeedbackUI_API DamageFeedbackUI : public Script
+{
+public:
+	void Awake() override;
+	void Update() override;
+	void Expose(ImGuiContext * context) override;
```
Comment (AlfonsoMestres) [inline Scripts/DamageFeedbackUI/DamageFeedbackUI.h]: Codestyling

**Comment by AlfonsoMestres** [comment]
skeletonBug

The dmg indicator is being rendered OK but the skeleton is being stuck after the first hit.

File: Scripts/DamageFeedbackUI/DamageFeedbackUI.h (hunk 1 above)
Comment (AlbertVVila) [reply Scripts/DamageFeedbackUI/DamageFeedbackUI.h]: 👍

**Comment by AlbertVVila** [comment]
I changed the typo you pointed out but I couldn't reproduce that skeleton bug you mentioned. I will forward it to the enemy team

**Comment by AlfonsoMestres** [comment]
workingRedflash
It's working now, the UI damage indicator should fade faster. Imagine being hit by 4 enemies at once..

**Comment by AlbertVVila** [comment]
> 
> 
> workingRedflash
> It's working now, the UI damage indicator should fade faster. Imagine being hit by 4 enemies at once..

The red screen method has parameters for fadeIn and fadeOut that can be customized

**Pull request merged**